app/data_model/questionnaire_store.py

```diff
@@ -35,14 +33,14 @@ def save(self):
         logger.debug("Saving user data %s for user id %s", self.data, self.user_id)
         self.storage.store(data=self.data, user_id=self.user_id, user_ik=self.user_ik)
 
-    def encode_metadata(self, metadata):
-        self.data["METADATA"] = jsonpickle.encode(metadata)
-
-    def decode_metadata(self):
+    @property
+    def metadata(self):
         if "METADATA" in self.data:
-            return jsonpickle.decode(self.data["METADATA"])
-        else:
-            raise RuntimeError("No metadata for user %s", self.user_id)
```

Comment:
If this is removed; is the same error/message/check handled somewhere else (e.g. do all callers check for None as a return value ~ they may not have before etc. ); I.e if this occurs will it be easily identified in the log/stack trace.